firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -259,3 +278,85 @@ float getInstantaneousSlope(int radius, int xStepsRemaining, int yStepsRemaining
     
     return ((float) radius - xStepsRemaining) / yStepsRemaining;
 }
+
+// Moves upwards to the largest radius in full steps
+void moveToFirstCircle()
+{
+    // Loop counter
+    int i;
+
+    setScale(yMotor, WHOLE_STEPS);  
+    digitalWrite(yMotor[DIR_PIN], UP);  
+    for (i = 0; i < circleRadius[0]; i++)
+    {
+        // Y-Step
+        digitalWrite(yMotor[STEP_PIN], LOW);
+        digitalWrite(yMotor[STEP_PIN], HIGH);
+    }
+}
+
+void moveToNextCircle(int currentCircle)
```

Comment:
Can you please add a function comment explaining that after each circle is made, you proceed counter-clockwise to the next quadrant but also move in to the next smaller circle